stone/cli.py

```diff
@@ -104,6 +105,15 @@
           'combine multiple expressions with "and"/"or" and use parentheses '
           'to enforce precedence.'),
 )
+_cmdline_parser.add_argument(
+    '-r',
+    '--route-whitelist-filter',
+    type=six.text_type,
+    help=('Restrict datatype generation to only the routes specified in the whitelist '
+          'and their dependencies. Input should be a JSON file mapping namespace names to '
+          'either 1) a list of routes to generate, or 2) the character *, as shorthand '
+          'for generating all routes in that namespace.'),
+)
```

Comment:
Can we get either a comment or update the README with an example of what the whitelist file should look like.